.github/workflows/ci.yml

```diff
@@ -46,3 +46,34 @@ jobs:
           install-command: yarn --frozen-lockfile
       - name: Yarn build
         run: yarn build
+  lint-test-auth:
+    name: Lint and test auth package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # This clones the repo and checks out the SHA that triggered this action.
+          # We set fetch-depth 0 to fetch all branches and history so that merge-base
+          # is guaranteed to be able to find the common ancestor with the base branch.
+          # See https://github.com/actions/checkout#readme
+          fetch-depth: 0
+      - name: Yarn caching
+        uses: actions/cache@v2
+        id: yarn-caching
+        with:
+          path: "**/node_modules"
+          key: yarn-v2-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            yarn-v2-${{ hashFiles('**/yarn.lock') }}
+            yarn-v2-
+            yarn-
+      - name: Yarn install
+        if: steps.yarn-caching.outputs.cache-hit != 'true'
+        working-directory: packages/auth
+        run: yarn install --frozen-lockfile
+      - name: Yarn lint
+        working-directory: packages/auth
+        run: yarn lint
+      - name: Yarn test
+        working-directory: packages/auth
+        run: yarn test
```

Comment:
Is there a good way to test these changes? This repo's CI steps are slightly different to recidiviz-data's repo - I ultimately went with the recidiviz-data repo steps.

Reply:
I realized I didn't have the linter working! Now that it is, it gave me some action items. 😄

Reply:
no, push and pray is the only option. luckily that is your favorite emoji

Reply:
🤣 😆 🤣 😆  I'm absolutely cackling at this. I love it!